modules/services/utils/src/test/java/au/com/shiftyjelly/pocketcasts/utils/SharingUrlTimestampParserTest.kt

```diff
@@ -0,0 +1,33 @@
+package au.com.shiftyjelly.pocketcasts.utils
+
+import kotlin.time.Duration.Companion.seconds
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SharingUrlTimestampParserTest {
+
+    private val parser = SharingUrlTimestampParser()
+
+    @Test
+    fun parseTimestampInterval() {
+        assertEquals(Pair(28.seconds, 60.seconds), parser.parseTimestamp("28,60"))
+        assertEquals(Pair(0.seconds, 20.seconds), parser.parseTimestamp(",20"))
+        assertEquals(Pair(10.seconds, 0.seconds), parser.parseTimestamp("10"))
+    }
+
+    @Test
+    fun testParseTimestampHoursMinutesSeconds() {
+        assertEquals(Pair(454070.seconds, null), parser.parseTimestamp("125h67m50s"))
```

Comment:
This isn't correct. Something like `67m` shouldn't have passed. Minutes and seconds must be between `0` and `59`.

Reply:
[ad8b83a](https://github.com/Automattic/pocket-casts-android/pull/2235/commits/ad8b83a6190469c06444789215e405526dffcbca)